docs/tutorials/google/visualizing_calibration_metrics.ipynb

```diff
@@ -68,27 +68,29 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "9e4d953fcc28"
+    "cellView": "form",
+    "id": "T7hnERotRv12"
    },
    "outputs": [],
    "source": [
+    "#@title Necessary Imports\n",
     "try:\n",
     "    import cirq\n",
     "except ImportError:\n",
     "    print(\"installing cirq...\")\n",
-    "    !pip install --quiet cirq\n",
+    "    !pip install --quiet --pre cirq\n",
```

Comment:
If you use `--pre`, I think you need to add this to `NOTEBOOKS_DEPENDING_ON_UNRELEASED_FEATURES` in `dev_tools/notebooks/isolated_notebook_test.py`.

Reply:
done. Thanks for being vigilant!